snapshots/btrfs/btrfs_test.go

```diff
@@ -44,7 +45,10 @@ func boltSnapshotter(t *testing.T) func(context.Context, string) (snapshots.Snap
 		t.Skipf("could not find mkfs.btrfs: %v", err)
 	}
 
-	// TODO: Check for btrfs in /proc/module and skip if not loaded
+	procModules, err := ioutil.ReadFile("/proc/modules")
+	if err == nil && !bytes.Contains(procModules, []byte("btrfs")) {
```

Comment:
Considering `procModules` is already a `string`, why use `bytes.Contains` instead of `strings.Contains`?

Reply:
Readfile function returns a []byte type hence thought of using it like that. Could also convert it to a string and then use strings.Contain() but both approaches look similar.

Reply:
Thanks, I misread the code.